frontend/src/apiAccess/apis.js

```diff
@@ -405,6 +410,11 @@ async function activityLogEntries({
   if (object1Id) params["object_1_id"] = object1Id;
   if (object2ContentType) params["object_2_content_type"] = object2ContentType;
   if (object2Id) params["object_2_id"] = object2Id;
+  if (timestamp) params["timestamp"] = timestamp;
+  if (actorUserEmail) params["actor_user_email"] = actorUserEmail;
+  if (effectedUserEmail) params["effected_user_email"] = effectedUserEmail;
+  if (object1Summary) params["object_1_summary"] = object1Summary;
+  if (object2Summary) params["object_2_summary"] = object2Summary;
```

Comment:
please double check what filters are implemented on the backend. search for `class ActivityLogEntryViewSet`